integreat_compass/cms/models/offers/offer_version.py

```diff
@@ -60,6 +60,11 @@ class OfferVersion(AbstractBaseModel):
         verbose_name=_("Language"),
         help_text=_("The language being taught in this offer"),
     )
+    state = models.CharField(
```

Comment:
```suggestion
    state = models.BooleanField(
```
The data types of the underlying constants are Boolean, so either change the field type, or the type of the constants.